shared/checks/oval/installed_OS_is_sle15.xml

```diff
@@ -6,9 +6,9 @@
       <affected family="unix">
         <platform>multi_platform_all</platform>
       </affected>
-      <reference ref_id="cpe:/o:suse:linux_enterprise_server:15"
+      <reference ref_id="cpe:/o:suse:suse_linux_enterprise_server:15"
       source="CPE" />
-      <reference ref_id="cpe:/o:suse:linux_enterprise_desktop:15"
+      <reference ref_id="cpe:/o:suse:suse_linux_enterprise_desktop:15"
```

Comment:
This CPE change is incorrect. Please keep the original.

Reply:
Fixed